app/db/setup.sh

```diff
@@ -6,6 +6,8 @@ else
     db="realm_registry"
 fi
 
-echo "SELECT 'CREATE DATABASE $db' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = '$db')\gexec" | psql -d postgres
+postgres_username=<<postgres_username>>
```

Comment:
Hm, the only thing with this is this file isn't gitignored, so setting it in the file will potentially lead to us pushing over eachothers. IDK though this script is only for local so don't want to burn too much time on it. Maybe just explicitly setting to postgres is okay, or passing it in as a script arg